### PR TITLE
Adding a new page.

### PR DESCRIPTION
**Pull request by sahaaakanksha7**

### CODE REVIEW

**Comment by Anvesha4** [comment]
Changes look good. Lmk if you plan on adding the answer key as well

File: content/english/security/5-summary/_index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+1. At least 8 characters - the more characters, the better
+2. A mixture of letters and numbers
+3. Inclusion of at least one special character, e.g., ! @ # ? ]
+*Note: do not use < or > in your password, as both can cause problems in Web browsers*
+
+A strong password is hard to guess, but it should be easy for you to remember - a password that has to be written down is not strong, no matter how many of the above characteristics are employed.
+
+
+### Solution
```
[the remark this answers is not in the document]
Comment (sahaaakanksha7) [reply content/english/security/5-summary/_index.md:19]: Yeah before giving the answers I was hoping to reiterate on the qualities of the strong password and how to figure out one. It is based on previous activity and then I wanted to drive that why in the previous activity the following two passwords were weak.

[the remark this answers is not in the document]
Comment (sahaaakanksha7) [reply content/english/security/5-summary/_index.md:19]: This page is more of Summary + Solution

File: content/english/security/5-summary/_index.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+In the Spy Game the passwords "Ilovesoccer" and "12345678" are weak and can be easily hacked by a malicious person.
+
```
[the remark this answers is not in the document]
Comment (sahaaakanksha7) [reply content/english/security/5-summary/_index.md:22]: Yes I will add and answer key by EOD today :)

File: content/english/security/answer-key/_index.md [hunk 3]
```diff
@@ -1,25 +1,12 @@
 ---
 title: "Answer Key - Security"
 description: "Easy reference answer key for the security curriculum"
-draft: true
+draft: false
```
Comment (Anvesha4) [inline content/english/security/answer-key/_index.md:4]: let's keep it true so that it doesn't show up in the main workshop

**Pull request merged**